pkg/spec/cluster.go

```diff
@@ -193,6 +193,8 @@ type ClusterStatus struct {
 
 	// Size is the current size of the cluster
 	Size int `json:"size"`
+	// HealthySize is the number of pods passing the liveliness probe, HTTP GET on port 2379
```

Comment:
@hongchaodeng Why do we want to show endpoints here at all? As a user, why do i care about endpoint? All I want is to see the pod name, so i can inspect it. I can get the endpoint myself to query pod.